feat(districts): District admin create-page

npthang opened the pull request:
see #257

npthang committed:
feat(districts): District admin create-page

see #257

huytbt commented on app/Http/Controllers/Admin/DistrictController.php:
```diff
@@ -13,4 +14,21 @@ public function index()
+    {
+        $this->validate($request, [
+            'name' => 'required|unique:districts|max:255',
+        ]);
```
Need validate city_id, require, exists, numeric

huytbt commented on resources/views/admin/districts/create.blade.php:
```diff
@@ -0,0 +1,39 @@
+    <div class="row">
+        <div class="col-md-12">
+            <div class="panel panel-default">
+                <div class="panel-heading">Create Districts</div>
```
Create District. not Districts

npthang committed:
feat(districts): District admin create-page

modified:   app/Http/Controllers/Admin/DistrictController.php
modified:   resources/views/admin/districts/create.blade.php

see #257

npthang committed:
feat(districts): Districts admin create-page

see #257

huytbt reviewed (approved):
ok

huytbt committed:
Merge branch 'master' into feat/districts/create-page
